apps/ledger-live-mobile/src/screens/ReceiveFunds/03a-ConnectDevice.tsx

```diff
@@ -102,6 +102,7 @@ export default function ConnectDevice({ navigation, route }: Props) {
   }
 
   const mainAccount = getMainAccount(account, parentAccount);
+  const currency = mainAccount && getAccountCurrency(mainAccount)
```

Comment:
`getMainAccount` will throw if there is no `mainAccount` so the check is a bit redundant IMO